lib/lbt/analyzer/JSModuleAnalyzer.js

```diff
@@ -398,7 +398,7 @@ class JSModuleAnalyzer {
 					 */
 				} else if ( nModuleDeclarations > 1 && name === info.name ) {
 					// ignore duplicate declarations (e.g. in behavior file of design time controls)
-					log.warn("duplicate declaration of module name at %s", getLocation(args));
+					log.warn("duplicate declaration of module "+ `${getLocation(args)}`+ " in "+ `${name}`);
```

Comment:
When you start using es6 string templates, why don't you make the whole line a single template?

  `duplicate declaration of module ${getLocation(args)} in ${name}`

Reply:
So true...